app/views/visit/_cancellation_form.html.haml

```diff
@@ -3,9 +3,9 @@
   .group.options
 
     %label.block-label(for="cancel_booking")
-      %input#cancel_booking{ type: "checkbox", value: "#{type}", name: "cancel" }
+      %input#cancel_booking{ type: "checkbox", value: true, name: "cancel" }
       = hidden_field_tag :state, @state
       Yes, I want to cancel this visit
 
   .actions
-    %input.button.button-primary{ type: "submit", value: "Cancel #{type}" }
+    %input.button.button-primary{ type: "submit", value: "Cancel" }
```

Comment:
The heading above the form does mention 'request' when appropriate. This PR is not a design change, so I'd prefer to retain this as previously designed.